package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "protocol",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Comment:
If you would like to publish this straight away, then you'd need to follow [these steps](https://github.com/mozilla/protocol/tree/master/docs#publishing) in addition to updating the version number here.

If not, just adding to the changelog under `HEAD` is fine, and we can update the version number when ready to next publish.

Reply:
> This could conflict with sites using Protocol in production so this also bumps the version to 2.2.1.

If this is something that could conflict in production, we should also make it a major version bump according to https://semver.org/. I'm not sure what is a breaking change here though (unless I'm missing something).

Reply:
There are a few pages in bedrock with these same rules applied to buttons that override the older Protocol style, so I don't *think* this will cause major breakage but would at least make those overriding styles redundant. I can  do some testing to assess the potential for damage.

Reply:
Ah, makes sense - then yes I'd say at least make this `2.3.0`. A patch version change would be picked up and installed via NPM automatically, which is probably not what we'd want here.

Reply:
Ah didn't realize minor versions were published automatically. I'll update.